server/v2/appmanager/config.go

```diff
@@ -2,7 +2,7 @@ package appmanager
 
 // Config represents the configuration options for the app manager.
 type Config struct {
-	ValidateTxGasLimit uint64 `mapstructure:"validate-tx-gas-limit"` // TODO: check how this works on app mempool
+	ValidateTxGasLimit uint64 `mapstructure:"validate-tx-gas-limit"`
```

Comment:
i believe this can be removed, we should open an issue on it to track it.

Reply:
It's being used here actually: https://github.com/cosmos/cosmos-sdk/blob/3860b2b04b964106f4b7430ab56289447cc11768/server/v2/appmanager/appmanager.go#L235